src/components/Molecules/JoinPoll/AnimalPicker/AnimalPicker.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {
+  StyleSheet,
+  View,
+} from 'react-native'
+
+import {
+  ColoredCircle,
+} from '../../../../components'
+import gStyles from '../../../styles'
+
+// const { height, width } = Dimensions.get('window')
```

Comment:
I don't like commented out code - do we still need this?

Reply:
👍